### DISCUSSION TITLE
Notice Subtype 1-2 -> only Proc level?

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#711] [Q&A] [answered]
Hello Lab,

We have to implement notice subtype 1-2 as well, and now we see that there are a lot of fields what we cannot use on lot level only procedure level. 
Example: BT-21-Procure (but we cannot use BT-21-Lot) or fields of preformance place (eg: BT-5071-Procedure but not BT-5071-Lot).

This is the exact opposite of what we have seen in the other notices, because what we have given so far at the part level, now we have to give it at the procedure level.

Is this mean we cannot use more lots in notice subtypes 1 and 2?
Or am I wrong?

Thanks is advance
Anna

**Comment by YvesJo** [comment, marked as answer]
Hi,
Please get in touch with the TED helpdesk ted@publications.europa.eu for this non-technical question.
